src/test/resources/feature/BDDscenarioForWishList.feature

```diff
@@ -0,0 +1,46 @@
+@WishList
+Feature: Wish List;
+
+  Background:
+    Given user is on http://automationpractice.com/ page
+
+
+  @CreateWishList
+  Scenario Outline: Create Wish List
+    Given user is logged in
+    And user is on MY WISHLISTS page
+    And user type on name field "<name>"
+    When user click on Save button
+    Then Wishlist is created
+
+    Examples:
+      | name         |
+      | My favourite |
+      | 1235         |
```

Comment:
Why do you need to run this test 3 times?

Reply:
|12345| - removed